translations/de/info/task_description.html

```diff
@@ -1,66 +1,78 @@
+<html>
+    <meta charset="utf-8"
+</html>
 <div style="text-align: center">
         <img src="{{MEDIA}}title.png" style="max-height: 350px; max-width: 100%">
 </div>
 
 <p>
-    Roman numerals come from the ancient Roman numbering system.
-    They are based on specific letters of the alphabet which are combined to signify the sum
-    (or, in some cases, the difference) of their values. The first ten Roman
-    numerals are:
+    Die römische Zahlen stammen aus dem alten römischen Zahlensystem.
+    Sie basieren auf bestimmten Buchstaben des Alphabets.
+    Addiert man die Werte der römischen Ziffern, ergibt die Summe (Oder,
+    in einigen Fällen die Differenz) die Zahl.
```

Comment:
I **do not** like commas, I just follow the recommendations of Duden. :smile: 
:+1: